wallet/src/main/res/values/strings.xml

```diff
@@ -357,6 +357,14 @@
     <!-- External browser -->
     <string name="browser_continue">Launching browser, continue there…</string>
 
+    <!-- Presentation while issuing evidence -->
+    <string name="presentation_evidence_message">
```

Comment:
minor cleanup:
The Issuing Authority requires you to authenticate yourself.
You can do this by presenting the eID you have in your wallet.